.gitignore

```diff
@@ -5,6 +5,8 @@
 *.so
 *.dylib
 
+main.go
```

Comment:
🤔

Reply:
Added to restrict the main file written for testing   our publisher not added in repo